lib/Extractor/Candidates.cpp

```diff
@@ -37,6 +37,7 @@
 #include <unordered_set>
 #include <tuple>
 #include "llvm/Analysis/ValueTracking.h"
+#include "llvm/Analysis/TargetLibraryInfo.h"
```

Comment:
Random comment - usually we try to keep alphabetical order for the including header files.

Reply:
Thanks for the tip! This include is removed, since TargetLibraryInfo.h is already included in Candidates.h :-)